### PR TITLE
Add three recipes for asdf schema

### PR DESCRIPTION
**Pull request by mwcraig**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/asdf-wcs-schemas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - asdf_wcs_schemas
+    - asdf_wcs_schemas.resources
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/spacetelescope/asdf-wcs-schemas
```
Comment (dopplershift) [inline recipes/asdf-wcs-schemas/meta.yaml:38]: Typo:
```suggestion
  home: https://github.com/asdf-format/asdf-wcs-schemas
```

File: recipes/asdf-transform-schemas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - setuptools-scm
+  run:
+    - asdf >=2.8.0
+    - importlib_resources >=3  # [py<39]
```
Comment (dopplershift) [inline recipes/asdf-transform-schemas/meta.yaml:25]: Would be better to just unconditionally depend on `importlib_resources` and make the recipe noarch.

File: recipes/asdf-wcs-schemas/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - setuptools-scm
+  run:
+    - asdf >=2.8.0
+    - importlib_resources >=3  # [py<39]
```
Comment (dopplershift) [inline recipes/asdf-wcs-schemas/meta.yaml:25]: Would be better to just unconditionally depend on `importlib_resources` and make the recipe noarch.

Comment (mwcraig) [reply recipes/asdf-wcs-schemas/meta.yaml]: Yes, done.

File: recipes/asdf-transform-schemas/meta.yaml (hunk 2 above)
Comment (mwcraig) [reply recipes/asdf-transform-schemas/meta.yaml]: Yes, done.

**Comment by mwcraig** [comment]
@conda-forge/staged-recipes -- this is another old-ish PR ready for re-review now.

**Comment by beenje** [comment]
Looks good to me.
There is a version 0.2.2 for asdf-transform-schemas, but as this PR is quite old, I'll merge.
Recipe can be updated after.

**Pull request merged**